test/fixtures/test-runner/output/stdout.js

```diff
@@ -0,0 +1,9 @@
+// Flags: --no-warnings
+'use strict';
+require('../../../common');
+const test = require('node:test');
+
+test('ok', () => {
```

Comment:
there are already tests testing this, I don't think we need another one:

https://github.com/nodejs/node/blob/6e503cabd8b2e71472b84cbff6f5c1b0d3feef5e/test/fixtures/test-runner/user-logs.js
https://github.com/nodejs/node/blob/6e503cabd8b2e71472b84cbff6f5c1b0d3feef5e/test/fixtures/test-runner/output/arbitrary-output.js